types/bech32/legacybech32/pk_bench_test.go

```diff
@@ -1,14 +1,12 @@
-package types_test
+package legacybech32
 
 import (
 	"math/rand"
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/require"
-
 	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
-	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
```

Comment:
I think it may help to run `make format` here (I'd suggest to use the git hook available in `contrib`)

Reply:
Linter doesn't handle that (doesn't report this as an error). I fixed it. Thanks for catching.